custom_components/bermuda/coordinator.py

```diff
@@ -523,11 +523,24 @@ async def _async_update_data(self):
                 device.local_name = clean_charbuf(service_info.advertisement.local_name)
 
             device.manufacturer = device.manufacturer or service_info.manufacturer
+            if device.manufacturer is None:
+                if (
+                    service_info.service_uuids
+                    and "feed" in service_info.service_uuids[0]
+                    or "feec" in service_info.service_uuids[0]
+                ):
+                    # https://bitbucket.org/bluetooth-SIG/public/src/main/assigned_numbers/uuids/member_uuids.yaml
+                    device.manufacturer = "Tile"
             device.connectable = service_info.connectable
 
             # Try to make a nice name for prefname.
             if device.prefname is None or device.prefname.startswith(DOMAIN + "_"):
-                device.prefname = device.name or device.local_name or DOMAIN + "_" + slugify(device.address)
+                device.prefname = (
+                    device.name
+                    or device.local_name
+                    or f"{DOMAIN}_{slugify(device.address)}{f" ({slugify(device.manufacturer)}) "
+                if device.manufacturer else ""}"
```

Comment:
This is a me thing not a you thing, but I am having very strong visceral reactions to this ternary! Can you re-write it as a classic if-then-else? I keep trying to parse it and just get frustrated before getting anywhere 😅 I can't tell if it does what I expect because the surrounding indentation and quoting and parens make the usually smelly ternary thoroughly opaque to me! It might be an adhd limited working-set-memory thing, but I just don't have the stack space in my own brain for it, sorry! I feel like beyond a single line it's an antipattern.

While there... what do you think about ditching the `{DOMAIN}_` part and replacing that with the slugified manu name as well? eg: `tile_44_55_11_22_33` which is probably more helpful for users. I only put the domain in there because I didn't want to create entries with just a bare mac address, so having the manufacturer name prefixed (when available) I think is a better user-experience.

Reply:
Yeah replacing domain is a good call. 

I removed the ternary.

I also added the whole list of member uuids. Maybe in the future we could setup a github action to automatically check for updates like once a week?